add async trigger for `hubspot` (#72)
ballerina-platform/asyncapi-triggers: asyncapi/hubspot/Package.md

> Listen to [Hubspot event API](https://developers.hubspot.com/docs/api/webhooks/) from Ballerina

Reviewer: ```suggestion
Listen to [HubSpot event API](https://developers.hubspot.com/docs/api/webhooks/) from Ballerina.
```

Author: @LahiruRajapaksha Shall I change as suggested? or do we have a convention on this wording? I mean in all triggers, we have used like `Listen to`

Reviewer 2: @Drifftr requesting your feedback on this

Author: Changed as discussed with @RukshiW

## Package overview

> The `ballerinax/trigger.hubspot` is a [Ballerina](https://ballerina.io/) trigger for Hubspot events API.

Reviewer: Change "Hubspot" to "HubSpot".

Author: Changed as suggested.

> This package provides the capability to access Hubspot Webhook API.

Reviewer: ```suggestion
This package allows you to access HubSpot Webhook API.
```

Author: Changed
